.github/workflows/CI-unixish.yml

```diff
@@ -553,12 +553,10 @@ jobs:
           # self check simplecpp
           ./cppcheck $selfcheck_options externals/simplecpp || ec=1
           # self check lib/cli
-          mkdir b1
-          ./cppcheck $selfcheck_options $cppcheck_options --cppcheck-build-dir=b1 --addon=naming.json cli || ec=1
-          ./cppcheck $selfcheck_options $cppcheck_options --cppcheck-build-dir=b1 --addon=naming.json --enable=internal lib || ec=1
+          ./cppcheck $selfcheck_options $cppcheck_options --addon=naming.json cli || ec=1
```

Comment:
this skips the ctu checkers.

Reply:
I was not aware of that. 

But that kind of supports the case that it seems "random" and we need to properly integrate it. So we should probably also add it in daca.

I assume that we also lack test coverage for this.